### PR TITLE
net-irc/znc: bump to 1.7.0 + HOMEPAGE update

### PR DESCRIPTION
**Pull request by sbraz**
@DarthGandalf let me know if you see anything wrong with this.

Changelog [here](https://github.com/znc/znc/blob/znc-1.7.0/ChangeLog.md).

This is mostly taking the changes from the live ebuild and creating a new 1.7.0 ebuild with them. I've also updated HOMEPAGE in all ebuilds so that it's reflected on packages.gentoo.org.

I don't know why gtest was not present in the Manifest before, it was already required by the live ebuild.

### CODE REVIEW

**Comment by DarthGandalf** [comment]
LGTM, thanks.
Maybe because gtest is optional?

File: net-irc/znc/znc-1.7.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+inherit cmake-utils python-single-r1 readme.gentoo-r1 systemd user
+
+GTEST_VER="1.8.0"
+GTEST_URL="https://github.com/google/googletest/archive/release-${GTEST_VER}.tar.gz -> gtest-${GTEST_VER}.tar.gz"
+DESCRIPTION="An advanced IRC Bouncer"
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI=${EGIT_REPO_URI:-"https://github.com/znc/znc.git"}
+	SRC_URI=""
```
Comment (mgorny) [inline net-irc/znc/znc-1.7.0.ebuild]: Don't you need gtest in -9999? Or is it included in the repo?

File: net-irc/znc/znc-1.7.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+		export GTEST_ROOT="${WORKDIR}/googletest-release-${GTEST_VER}/googletest"
+		export GMOCK_ROOT="${WORKDIR}/googletest-release-${GTEST_VER}/googlemock"
+	fi
+
+	cmake-utils_src_configure
+}
+
+src_test() {
+	pushd "${BUILD_DIR}" > /dev/null || die
+	${CMAKE_MAKEFILE_GENERATOR} unittest || die "Unit test failed"
```
Comment (mgorny) [inline net-irc/znc/znc-1.7.0.ebuild]: `cmake-utils_src_make`. I think it also takes care of changing directory.

File: net-irc/znc/znc-1.7.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+	if [[ -e "${EROOT%/}/var/lib/znc" ]]; then
+		ewarn "${EROOT%/}/var/lib/znc already exists, aborting to avoid damaging"
+		ewarn "any existing configuration. If you are sure you want"
+		ewarn "to generate a new configuration, remove the folder"
+		ewarn "and try again."
+	else
+		einfo "Press any key to interactively create a new configuration file"
+		einfo "for znc."
+		einfo "To abort, press Control-C"
+		read
```
Comment (mgorny) [inline net-irc/znc/znc-1.7.0.ebuild]: 'Any key' won't work here ;-).

File: net-irc/znc/znc-1.7.0.ebuild (hunk 1 above)
Comment (sbraz) [reply net-irc/znc/znc-1.7.0.ebuild]: It's in a submodule https://github.com/znc/znc/tree/master/third_party

**Comment by sbraz** [comment]
@mgorny I addressed the issues you mentioned and reworded README.gentoo a little bit.

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**